.github/workflows/build-and-test.yml

```diff
@@ -259,6 +259,12 @@ jobs:
         name: cmsis-stm32f4xx-build-artifact
         path: build-cmsis/samples
 
+    - name: Download Libopencm3 build artifact for STM32F4xx
```

Comment:
```suggestion
    - name: Download libopencm3 build artifact for STM32F4xx
```

Reply:
Reasonable suggestion. I was confident enough to apply it ;)